kickstart/iPXE.erb

```diff
@@ -26,7 +26,11 @@ oses:
 <%# This template will not function with Safemode set to true.
     Please disable it in Settings > Provisioning               %>
 
+<% if (@host.operatingsystem.name == 'Fedora' and @host.operatingsystem.major.to_i > 16) or (@host.operatingsystem.name != 'Fedora' and @host.operatingsystem.major.to_i >= 7) -%>
+kernel <%= "#{@host.url_for_boot(:kernel)}" %> inst.ks=<%= foreman_url('provision')%><%= static %> bootif=<%= @host.mac %> network inst.ks.sendmac ip=${netX/ip}::${netX/gateway}::${netX/netmask}:::none nameserver=${dns}
```

Comment:
Hmm `bootif` seems to be still supported: https://github.com/rhinstaller/anaconda/blob/master/dracut/parse-anaconda-net.sh#L39

Removed `network`